__tests__/app.test.js

```diff
@@ -178,3 +178,25 @@ describe("DELETE /api/comments/:comment_id", () => {
     return request(app).delete("/api/comments/2").expect(204);
   });
 });
+
+
+
+describe("GET /api/users", () => {
+  test("gives status of 200 and responds with all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { allUsers } }) => {
+        allUsers.forEach((user) => {
```

Comment:
You need to check the length of the array first. If you use a forEach on an empty array, it will not run so your assertions will be skipped over.